Makefile

```
@@ -18,7 +18,11 @@ BUILDTAGS ?= seccomp urfave_cli_no_docs
 BUILDTAGS += $(EXTRA_BUILDTAGS)
 
 COMMIT ?= $(shell git describe --dirty --long --always)
-VERSION ?= $(shell cat ./VERSION)
+VERSION := $(shell cat ./VERSION)
+VERSION_EXTRA :=
+ifneq (,$(VERSION_EXTRA))
+	VERSION := $(VERSION)-$(VERSION_EXTRA)
```

```suggestion
	VERSION := $(VERSION)$(VERSION_EXTRA)
```
we should not be having the `-`, it will conflict with semver

---

Heh, I realized before checking this. I added a `+` sign now, that should be according to semver. Is it better for you to not add anything at all?

---

If you are building a dev version it already has a +. So let's just concat and warn users about the separator

---

>  Is it better for you to not add anything at all?
Yes, That would be better

---

Thanks!